src/js/view/preview.js

```diff
@@ -1,13 +1,39 @@
 define([
-    'utils/underscore'
-], function(_) {
+    'utils/underscore',
+    'utils/helpers'
+], function(_, util) {
 
     var Preview = function(_model) {
         this.model = _model;
 
-        this.model.on('change:playlistItem', this.loadImage, this);
+        this.model.on('change:playlistItem', onPlaylistItem, this);
     };
 
+    function loadOnComplete() {
+        if (this.model.get('state') === 'complete') {
+            this.loadImage(this.model, this.model.get('playlistItem'));
+        }
+    }
+
+    function onPlaylistItem(model, playlistItem) {
+        var audio = (this.model.mediaModel.get('mediaType') === 'audio');
+        var idleManualStart = (this.model.get('state') === 'idle') && (!this.model.get('autostart'));
+        var idleMobile = (this.model.get('state') === 'idle') && (util.isMobile());
+        var endOfPlaylist = (this.model.get('item') === (this.model.get('playlist').length - 1));
+
+        this.model.off('change:state', loadOnComplete);
+        if (audio || idleManualStart || idleMobile) {
+            // load image if audio mode, or mobile/non-autostart with on idle state
+            this.loadImage(model, playlistItem);
+        } else {
+            this.el.style.backgroundImage = '';
```

Comment:
If we do not clear this, it would show the previous preview image just before buffering if the previous file was audio. Is there another way to handle this?